lib/bazel/commands/commands.go

```diff
@@ -103,6 +105,9 @@ func NewAffectedTargetsList(parent *AffectedTargets) *AffectedTargetsList {
 	command.Flags().BoolVar(&command.Parallel, "parallel", true, "If set, the bazel query is run in parallel")
 	command.Flags().StringVar(&command.AffectedTargetsFile, "affected_targets_file", "", "If set, the list of affected targets will be dumped to this file path")
 	command.Flags().StringVar(&command.AffectedTestsFile, "affected_tests_file", "", "If set, the list of affected tests will be dumped to this file path")
+	command.Flags().StringVar(&command.Start.OutputBase, "start_output_base", "", "If set, the directory to use as start output base")
+	command.Flags().StringVar(&command.End.OutputBase, "end_output_base", "", "If set, the directory to use as end output base")
+	command.Flags().StringVar(&command.Query, "query", "deps(//...)", "The query to use to find the targets")
```

Comment:
I doubt the rest of the logic can tolerate arbitrary queries - maybe a caveat on this flag?

Reply:
The reason I added it is 1) so I could test with a query that was faster, 2) I think the logic will work ok for /some/ queries. For example, a team could add a faster presubmit for only their subdirectory? or for eg, the website push, I could constrain to rdeps of web targets? And compute the delta there? Adding a comment.